server/src/test/java/org/opensearch/common/blobstore/fs/FsBlobContainerTests.java

```diff
@@ -34,6 +34,9 @@
 import org.apache.lucene.tests.mockfile.FilterFileSystemProvider;
 import org.apache.lucene.tests.mockfile.FilterSeekableByteChannel;
 import org.apache.lucene.tests.util.LuceneTestCase;
+import org.opensearch.action.ActionListener;
```

Comment:
Changes in this file will be added as part of https://github.com/opensearch-project/OpenSearch/pull/8314. Reviewers can ignore these changes in this PR.